foundations/src/telemetry/log/init.rs

```diff
@@ -95,12 +96,29 @@ pub(crate) fn init(service_info: &ServiceInfo, settings: &LoggingSettings) -> Bo
         }
         (LogOutput::File(file), LogFormat::Json) => {
             let buf = BufWriter::with_capacity(BUF_SIZE, File::create(file)?);
-            let drain = build_json_log_drain(buf);
+            let drain = build_json_log_drain(buf, false);
             AsyncDrain::new(drain).chan_size(CHANNEL_SIZE).build()
         }
+        (LogOutput::Stdout, log_fmt) => {
+            let stdout = unsafe { File::from_raw_fd(STDOUT_FILENO) };
```

Comment:
Let's refactor this unsafe expression into a function, e.g. `unbuffered_stdout()` or something like that and add a code comment explaining why we need it this way